features/old/buyers/users.feature

```diff
@@ -57,7 +57,7 @@ Feature: Buyer users management
   Scenario: Delete buyer user
     When I go to the buyer user page for "bob"
     And I follow "Edit"
-    Then I follow "Delete" and I confirm dialog box
+    Then I follow invisible "Delete" and I confirm dialog box
```

Comment:
hmmm I don't think this is the way to go. Usually, a feature indicates what the user should see and interact with. If we are tampering a cuke step with clicking an invisible button/link, the feature is just a fallacy.

Reply:
The problem is I can't reproduce it because the test pass locally, I was trying this to see if this was the problem, but its still failing.